contrib/initramfs/zfsunlock

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -eu
```

Comment:
Personally, I'd put this in the shebang: #!/bin/sh -eu

If you prefer it here, that works too.